src/playground/Insights/ui/event.tsx

```diff
@@ -1,7 +1,7 @@
 import { cn } from "@/lib/utils"
 import { forwardRef } from "react"
 
-interface EventProps {
+export interface EventProps {
   title: string
   length: number
   lengthUnit: string
```

Comment:
I'm not sure if we should provide the subtitle in this way, this is indeed the most rigid way, but adding a 'subtitle' it's a more scalable. Let's keep this in that way, but I have doubts

Reply:
I changed to subtitle based on Arnau agreeing with this opinion too